addons/website/models/website_page.py

```diff
@@ -175,8 +175,10 @@ def unlink(self):
         return super(Page, self).unlink()
 
     def write(self, vals):
-        if 'url' in vals and not vals['url'].startswith('/'):
-            vals['url'] = '/' + vals['url']
+        if 'url' in vals:
+            url = vals['url'] or ''
+            if not url.startswith('/'):
+                vals['url'] = '/' + url
```

Comment:
assuming only a string or `False` can be sent and nothing else:
```suggestion
        if 'url' in vals and vals['url'] and not vals['url'].startswith('/'):
            vals['url'] = '/' + vals['url']
```

Reply:
Indeed, that's correct.
I pushed a diff to make it more explicit what we try to achieve, including a comment mentioning that the field is not required but we don't want to save empty url value so we fallback on /

But I actually don't want to merge it in 13.0. See the review message.